src/components/header/HeaderContainer.jsx

```diff
@@ -1,8 +1,15 @@
+// @flow strict
 import React from 'react';
-import PropTypes from 'prop-types';
+import type {Node} from 'react';
```

Comment:
`import * as React from 'react';`

Reply:
our intellisence doesn't recognize type of `Node` in that case

Reply:
but maybe you could do something like this? :
```
import * as React from 'react';
something: React.Node
```

Reply:
yeah, i meant exactly that - jsdoc based intellisense shows `any` instead of `Node` on hover. no idea why does it work like that. direct import works fine here

Reply:
Ah, ok, I get it now. 🙃